test/e2e/epp/e2e_suite_test.go

```diff
@@ -101,6 +102,11 @@ func TestAPIs(t *testing.T) {
 }
 
 var _ = ginkgo.BeforeSuite(func() {
+	nsName = os.Getenv("E2E_NS")
```

Comment:
Can you update the e2e readme so users know that `E2E_NS` can be used to change the default ns?